Getting started tutorial

amottier opened the pull request:
Totally refactor the getting started tutorial:
- split in several chapters
- up to date for Bonita 7.9
- step by step guidance
- includes short videos

Commit:
Getting started studio download and install

Commit:
Add what is Bonita chapter

Add the first chapter (optionnal) of the getting started tutorial.

Commit:
Add chapter to create BPMN diagram

Commit:
Add BDM definition and business variable declaration chapters

Commit:
Add contract chapter for getting started tutorial

Commit:
Add forms creation to getting started tutorial

Commit:
Add actor/actor filter chapter to getting started

Commit:
Add email connector configuration instructions

Commit:
Add page design and application creation chapters

Commit:
Review and improvement of 3 first chapters

Rephrase, add screenshots and videos.

Commit:
Fix layout and images URLs

Commit:
Rephrase and add screenshot for BDM definition

Commit:
Add gif for business variable declaration

Cwin-mkg committed:
Review Carole 1 (#2)

Update what-is-bonita.md

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/bonitasoft/bonita-doc?pullRequest=807) <br/>All committers have signed the CLA.

Commit:
Fix lists type

Ordered lists.

Commit:
Add instruction for gateway condition with business variable

dcoille committed:
Few english changes

Commit:
Update draw-bpmn-diagram.md

Commit:
Update draw-bpmn-diagram.md

Commit:
Merge branch 'dcoille-patch-2' into getting-started-tutorial

Update the file and remove comment.

Commit:
Add titles and todos

Need to fix todos before publishing.

Commit:
Merge remote-tracking branch 'carole/Carole-review-1' into getting-started-tutorial

Also include some information about the process example execution scenario and limitations of first process execution.

Commit:
Transition condition and contract definition

Explain configuration of transition using business data attribute with a gif.

Add gif and format contract definition.

Commit:
english edits

english edits

Commit:
english corrections

Commit:
Update in progress on actors

Commit:
Merge remote-tracking branch 'carole/Carole-review-1' into getting-started-tutorial

Commit:
Add missing link to next chapter

Commit:
Added links to other pages when introducing new concepts: gateways, cases
Added link to next chapter
Comment: the way to display INFO or WARNING does not really highlight it (at least in Git preview)

Commit:
English udpates
Link to next chapter

Commit:
English review and link added to next chapter

Commit:
English review and add link to next chapter

Commit:
Update web interface creation with gif

Also fix formatting

Commit:
Merge remote-tracking branch 'delphine/patch-2' into getting-started-tutorial

Commit:
Review and include screenshots for actors and connectors chapters

Commit:
Review and add screenshot application page creation

Commit:
Review and add screenshot to application creation chapter

Commit:
Update define-who-can-do-what.md

Commit:
Update configure-email-connector.md

Commit:
Update design-application-page.md

Commit:
Update create-application.md

Commit:
Merge remote-tracking branch 'carole/Carole-review-1' into getting-started-tutorial

Commit:
Add link to BPMN tutorial

As suggest by Carole.

Commit:
Add transition condition script explanation

Explain satisfaction level comparison.

Commit:
Mention that we are using Groovy as script language

For both condition and actor filter configuration (find task performer id).

Commit:
Merge remote-tracking branch 'upstream/7.9' into getting-started-tutorial
